local/vault/setup.sh

```diff
@@ -22,21 +22,21 @@ vault() {
 }
 
 message "Setting up Vault"
-#vault status
+sleep 5
```

Comment:
Would be great in the future to implement via liveness probes on docker-compose.